src/components/icons/subCategorizeOneIcon.tsx

```diff
@@ -0,0 +1,60 @@
+import { icon } from '../../../styled-system/recipes'
+import { token } from '../../../styled-system/tokens'
+import IconType from '../../@types/Icon'
+
+/** Bold icon. */
+const subCategorizeOneIcon = ({ fill = 'black', size = 20, style = {}, className }: IconType) => {
+  const scalingFactor = 1.37
+  const newSize = size * scalingFactor
+  const strokeColor = style.fill || fill || token('colors.fg')
```

Comment:
Yes, I'm open to that. Let's wait till after Milestone 2 of #2408 so that we can see the full extent of the common icon functionality including static styles and animation logic.